packages/config-schema/src/schema/netlify-toml.schema.json

```
@@ -0,0 +1,646 @@
+        "deno_import_map": {
+          "title": "Import maps",
+          "description": "Relative path to an import map file to map module URLs to names.",
+          "type": "string"
+        },
+        "external_node_modules": { "$ref": "#/definitions/FunctionConfigExternalNodeModules" },
+        "included_files": { "$ref": "#/definitions/FunctionConfigIncludedFiles" },
+        "node_bundler": { "$ref": "#/definitions/FunctionConfigNodeBundler" },
+        "directory": {
+          "title": "Directory",
+          "description": "Custom path to your functions. The default location is `YOUR_BASE_DIRECTORY/netlify/functions`.",
+          "type": "string",
+          "default": "netlify/functions"
+        }
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/configure-builds/file-based-configuration/#functions"
+        }
+      }
+    },
+    "edge_functions": {
+      "type": "array",
+      "default": [{ "path": "/path", "function": "function-name" }],
+      "items": {
+        "title": "Edge Functions",
+        "description": "Edge Functions connect the Netlify platform and workflow with an open runtime standard at the network edge. This enables you to build fast, personalized web experiences with an ecosystem of development tools.",
+        "type": "object",
+        "properties": {
+          "path": {
+            "type": "string",
+            "description": "The path pattern that needs to match for this edge function."
+          },
+          "function": {
+            "type": "string",
+            "description": "The name of the function."
+          }
+        },
+        "additionalProperties": false,
+        "required": ["path", "function"]
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/edge-functions/declarations/"
+        }
+      }
+    },
+    "dev": {
+      "title": "Netlify Dev",
+      "description": "Netlify Dev uses detectors to enable a local development environment for most tools and frameworks without any additional setup. You can use the `[dev]` section in `netlify.toml` for optional configuration.\n\nNote that `[dev]` doesn't run in the Bash shell, however, so you won't be able to use Bash-compatible syntax with the command.\n\nNetlify Dev also makes use of the functions directory setting to scaffold and serve your functions in a local development environment.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "command": {
+          "title": "Dev command",
+          "description": "A command that starts your development server or runs a command such as a compiler watch in the background. If no `targetPort` is specified, it runs the command in the background in addition to the static file server.",
+          "type": "string"
+        },
+        "port": {
+          "title": "Port",
+          "description": "The port that Netlify Dev is accessible from in the browser.",
```

Edit to be consistent with the others

```suggestion
          "description": "Port that Netlify Dev is accessible from in the browser.",
```